packages/jaeger-ui/src/actions/jaeger-api.js

```diff
@@ -15,11 +15,48 @@
 import { createAction } from 'redux-actions';
 import JaegerAPI from '../api/jaeger';
 
-export const fetchTrace = createAction(
-  '@JAEGER_API/FETCH_TRACE',
-  id => JaegerAPI.fetchTrace(id),
-  id => ({ id })
-);
+export function mergeAll(id) {
+  return traces => {
+    if (traces.data.length < 2) {
+      return traces;
+    }
+    const ot = { traceID: id, spans: [], processes: {}, warnings: [] };
```

Comment:
`ot` isn't the clearest name, maybe `multiTrace` would be clearer? I'd also move the following newline to precede this line. Also, I'm not sure what implications there may be from having an invalid `traceID` 🤔